### DISCUSSION TITLE
Is there any way to configure subpixel order?

### DISCUSSION THREAD

**Discussion by JordanP-Dev** [wezterm/wezterm#3422] [General]
My monitor has a BGR subpixel order, so RGB subpixel rendering doesn't look great.  Is there any way to configure this?

I found this commit mentioning blue/red subpixel flip: https://github.com/wez/wezterm/commit/d2419fb99e567e3b260980694cc840a1a3b86922.  Switching this back seems to make the subpixel rendering look good for me, but I'm wondering if there is a proper way to set it.

The below picture shows a comparison.  The top line is Wezterm with the blue / red commit reverted.  The second line is the current Wezterm main branch.  The third line is XFCE terminal.

![subpixel_rendering](https://user-images.githubusercontent.com/29902477/229305607-46015489-963d-4276-afa7-6ce6a7b54158.png)

**Comment by wez** [comment]
I just pushed 8582165ffcae924bae64306a5fd85e6f3f0f20fe to let you control this. Should show up in the nightly build within about 30 minutes or so.

```lua
config.display_pixel_geometry = "BGR"
```

**Comment by JordanP-Dev** [comment]
Wow, that was quick.  I've built it locally and it works great.  Thanks.
